src/app/pages/home/home.component.html

```diff
@@ -367,26 +297,46 @@ <h1 class="text-center title1">Talk To Us</h1>
           <br />
 
           <div class="col-md-8 mr-auto ml-auto">
-            <form action="mailto:sample@email.com?subject=Website Feedback" class="contact-form" enctype="text/plain"
-              method="POST">
+            <form
+              action="mailto:sample@email.com?subject=Website Feedback"
+              class="contact-form"
+              enctype="text/plain"
+              method="POST"
+            >
               <div class="row">
                 <div class="col-md-6">
                   <div class="input-group">
-                    <input class="form-control" placeholder="Name" type="text" />
+                    <input
+                      class="form-control"
+                      placeholder="Name"
+                      type="text"
+                    />
                   </div>
                 </div>
                 <div class="col-md-6">
                   <div class="input-group">
-                    <input class="form-control" placeholder="Email" type="email" />
+                    <input
+                      class="form-control"
+                      placeholder="Email"
+                      type="email"
+                    />
                   </div>
                 </div>
                 <br />
               </div>
-              <textarea class="form-control" placeholder="Tell us your thoughts and feelings..." rows="4"></textarea>
+              <textarea
+                class="form-control"
+                placeholder="Tell us your thoughts and feelings..."
+                rows="4"
+              ></textarea>
               <div class="row">
                 <div class="col-md-7 ml-auto">
                   <br />
-                  <button class="btn btn-warning" style="font-weight: 500" type="submit">
+                  <button
+                    class="btn btn-warning"
+                    style="font-weight: 500"
```

Comment:
Hey @Tevinthuku, It's important we use external files for all our styles, This will later help when we want to cache these styles over something like a CDN for first page loads.

Reply:
Done